### DISCUSSION TITLE
documentation of post navigation available (prev / next post)?

### DISCUSSION THREAD

**Discussion by 3muchfun** [GetPublii/Publii#1486] [Help] [answered]
Dear all, as the headline says, I'm looking for some documentation regarding building a post navigation? I did find anything in the documentation on website ...
Thanks, Michael

**Comment by candidexmedia** [comment]
What are you looking to do/hoping to achieve, and where are you getting stuck?

**Reply by 3muchfun** [reply]
nothing special, but I thought I could find something similar to https://getpublii.com/dev/paginationhbs-partial/ , but regarding posts. What possibilities do I have to build previous/next post navigation...

**Comment by bobmitro** [comment, marked as answer]
To navigate to the next or previous post, you should use the following tags:

`{{#nextPost}}` and `{{#previousPost}}` or  `{{#nextSimilarPost}}` and `{{#previousSimilarPost}}`

The difference is that the "Similar" tags navigate between posts from a specific tag. More details: https://getpublii.com/dev/post-page-tags/

Example: 
```
{{#checkIfAny ../previousPost ../nextPost}}
   <nav class="post__nav">
      {{#../previousPost}}
         <a href="{{url}}" rel="prev">{{title}}</a>
      {{/../previousPost}}
      {{#../nextPost}}
         <a href="{{url}}" rel="next">{{title}}</a>
      {{/../nextPost}}
   </nav>
{{/checkIfAny}}
```

**Reply by david-alexander-fleming** [reply]
Hi Bob, are `{{#nextSimilarPost}}` and `{{#previousSimilarPost}}` implemented differently to `#nextPost` and `#previousPost` ? I've adapted the example above but I'm having difficulty getting the following to render output. Thanks.

```
{{#post}}
    {{#checkIfAny ../previousSimilarPost ../nextSimilarPost}}
       <nav class="post__nav">
          {{#../previousSimilarPost}}
             <a href="{{url}}" rel="prev">{{title}}</a>
          {{/../previousSimilarPost}}
          {{#../nextSimilarPost}}
             <a href="{{url}}" rel="next">{{title}}</a>
          {{/../nextSimilarPost}}
       </nav>
    {{/checkIfAny}}
{{/post}}
```

**Comment by bobmitro** [comment]
Your code is on the right track. Have you checked the `renderer` section of your `config.json` file to enable similar post rendering?  https://getpublii.com/dev/how-to-configure-rendering-process-of-your-website/

`"renderSimilarPosts": true,`

**Comment by david-alexander-fleming** [comment]
That worked perfectly, Bob. I had it set to false - an oversight on my part, but I appreciate you taking the time to respond. Thank you!
